### ISSUE TITLE
Typo in evaluation/cross_validation.py

### ISSUE THREAD

**Issue by exe-devel** [frankkramer-lab/MIScnn#117]
There is a typo in /miscnn/evaluation/cross_validation.py  in line 89

88.         if any(isinstance(cb, ReduceLROnPlateau) for cb in cb_list):
89.             model.model.compile(optimizer=Adam(lr=model.learninig_rate),
90.                                 loss=model.loss, metrics=model.metrics)

FIX:

change "model.learninig_rate" to "model.learning_rate"
